history.rst

```diff
@@ -5,6 +5,37 @@
 OMERO version history
 =====================
 
+
+5.6.14 (November 2024)
+----------------------
+
+This release includes the following upgrade of the OMERO.server Java components:
+
+- omero-gateway-java 5.10.0
+- omero-blitz 5.8.0
+- omero-server 5.7.0
+- omero-renderer 5.6.0
+- omero-romio 5.7.0
+- omero-common 5.7.0
+- omero-model 5.7.0
+
+
+Improvements include:
+
+- Bump slf4j dependencies. See https://github.com/ome/openmicroscopy/pull/6409
+- Add new LUTs See https://github.com/ome/openmicroscopy/pull/6398 and https://github.com/ome/omero-web/pull/581
+- Ugrade of Bio-Formats to version 8.0.1
+- Fix dependency resolution following the upgrade of Bio-Formats see https://github.com/ome/openmicroscopy/pull/6413
+- Improve test coverage for table slice.  See https://github.com/ome/openmicroscopy/pull/6409
+
+**Note:** This version has been built via Java 11 with the compatibility flag on for Java 8.
+Limited testing was done with Java 8 but the application should still run on Java 8.
```

Comment:
This reads as "It is fine to run the application on Java 8" which is a different message than https://github.com/ome/omero-documentation/pull/2461